interface/src/main/java/xsbti/compile/ScalaInstance.java

```diff
@@ -17,16 +17,16 @@
 	/** A class loader providing access to the classes and resources in the library and compiler jars. */
 	ClassLoader loader();
 
-	/**@deprecated Only `jars` can be reliably provided for modularized Scala. (Since 0.13.0) */
-	@Deprecated
+	/**deprecated Only `jars` can be reliably provided for modularized Scala. (Since 0.13.0) */
+	// Deprecated
 	File libraryJar();
 
-	/**@deprecated Only `jars` can be reliably provided for modularized Scala. (Since 0.13.0) */
-	@Deprecated
+	/**deprecated Only `jars` can be reliably provided for modularized Scala. (Since 0.13.0) */
+	//Deprecated
 	File compilerJar();
 
-	/**@deprecated Only `jars` can be reliably provided for modularized Scala. (Since 0.13.0) */
-	@Deprecated
+	/**deprecated Only `jars` can be reliably provided for modularized Scala. (Since 0.13.0) */
+	//Deprecated
```

Comment:
These have been de-deprecated only so that I can compile the rest using `-Xfatal-warnings`. They have been deprecated for a while but are still used internally.